om.yaml

```diff
@@ -0,0 +1,85 @@
+# YAML version of https://github.com/juspay/omnix/blob/main/nix/modules/flake-parts/om.nix
```

Comment:
We should delete `om.nix` from the repo. Omnix expects users to use one or the other, not both (which doesn't make sense; and even if it did, it is error-prone to keep both in sync).

Reply:
https://github.com/juspay/omnix/pull/352/commits/be6de96c4d3d59cd24de2e3bc21367417e3d9f6d